lib/plugins/aws/lib/naming.js

```diff
@@ -0,0 +1,225 @@
+'use strict';
```

Comment:
Should this file live in `lib`? or should we move it to a `utils` directory as it's kind of a utility functionality...